go.mod

```diff
@@ -65,7 +65,7 @@ require (
 	github.com/openshift/client-go v0.0.0-20210409155308-a8e62c60e930
 	github.com/openshift/cloud-credential-operator v0.0.0-20200316201045-d10080b52c9e
 	github.com/openshift/cluster-api-provider-gcp v0.0.1-0.20201203141909-4dc702fd57a5
-	github.com/openshift/cluster-api-provider-ibmcloud v0.0.0-20210702173623-676faba9895d
+	github.com/openshift/cluster-api-provider-ibmcloud v0.0.0-20210925003227-c0f28b165bf7
```

Comment:
I don't see anything meaningful being brought in with this bump of the cluster api provider. How does this relate to the BZ?

Reply:
Reverted the update